crates/pera-core/src/dwallet_mpc/dkg.rs

```diff
@@ -41,6 +41,8 @@ pub(super) trait DKGSecondPartyPublicInputGenerator: Party {
 
 impl DKGFirstPartyPublicInputGenerator for DKGFirstParty {
     fn generate_public_input() -> DwalletMPCResult<MPCPublicInput> {
+        // TODO: Update hardcoded public parameters with valid values to Scaly's new version.
```

Comment:
```suggestion
        // Todo (Yael): Update hardcoded public parameters with valid values to Scaly's new version.
```

Just so I won't miss it